### PR TITLE
Play with PARTICIPANTS.md

### PR DESCRIPTION
**Pull request by luzengAdelaide**

### CODE REVIEW

File: PARTICIPANTS.md [hunk 1]
```diff
@@ -13,6 +13,6 @@ Person
 Person
 
 3
-
+# Thank you for the git workshop, I found it really useful.
```
Comment (kortschak) [inline PARTICIPANTS.md]: Can you please say when the workshop was.

**Comment by kortschak** [comment]
Please explain reason for PR in initial comment next time.

File: PARTICIPANTS.md (hunk 1 above)
Comment (luzengAdelaide) [reply PARTICIPANTS.md]: The workshop held on 27 March, 2018.

Comment (kortschak) [reply PARTICIPANTS.md]: In the file, please.

**Comment by luzengAdelaide** [comment]
PTL

**Comment by kortschak** [comment]
Thank you.

**Pull request merged**